### PR TITLE
Add site analytics

### PR DESCRIPTION
**Pull request by kabilar**
- [x] Create property on the DANDI Google Analytics account

### CODE REVIEW

**Comment by yarikoptic** [comment]
I don't mind.

File: mkdocs.yml [hunk 1]
```diff
@@ -64,6 +64,9 @@ plugins:
 # Customize theme
 extra:
   homepage: https://dandiarchive.org
+  analytics:
+    provider: google
+    property: G-15WQLCLQ3L
```
Comment (waxlamp) [inline mkdocs.yml:69]: It doesn't seem like a great idea to have this be in the public-facing code.

Comment (yarikoptic) [reply mkdocs.yml:69]: isn't that ID anyways present somewhere within HTML/JS sources of the page, thus in public facing metadata? yeap:

```
❯ wget -q -O- https://www.dandiarchive.org/handbook/ | grep G-15WQLCLQ3L
<script id="__analytics">function __md_analytics(){function n(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],n("js",new Date),n("config","G-15WQLCLQ3L"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){this.value&&n("event","search",{search_term:this.value})}),document$.subscribe(function(){var a=document.forms.feedback;if(void 0!==a)for(var e of a.querySelectorAll("[type=submit]"))e.addEventListener("click",function(e){e.preventDefault();var t=document.location.pathname,e=this.getAttribute("data-md-value");n("event","feedback",{page:t,data:e}),a.firstElementChild.disabled=!0;e=a.querySelector(".md-feedback__note [data-md-value='"+e+"']");e&&(e.hidden=!1)}),a.hidden=!1}),location$.subscribe(function(e){n("config","G-15WQLCLQ3L",{page_path:e.pathname})})});var e=document.createElement("script");e.async=!0,e.src="https://www.googletagmanager.com/gtag/js?id=G-15WQLCLQ3L",document.getElementById("__analytics").insertAdjacentElement("afterEnd",e)}</script>
```

so IMHO there is really no concern, or what is it?

Comment (kabilar) [reply mkdocs.yml:69]: Thanks team.  I have read that Google Analytics (GA) are set up in both ways.  That is, some keep the GA tag in the source code and others use an environment variable.  As Yarik mentioned, it is public in the HTML code.  The only concern I have seen on a Stackoverflow post (which I can't find now) is that keeping the GA tag in the source code makes it easier if someone wants to spam our GA metrics.

As a team we should decide on the strategy here and keep it consistent across our sites.  It looks like the Archive also has a Universal Analytics (legacy) tag in the [source code](https://github.com/dandi/dandi-archive/blob/6e4bf727a02985aceb0fb896736883c00929c142/web/src/main.ts#L50) and so does the [dandi.github.io repo](https://github.com/dandi/dandi.github.io/blob/1369aa4c51d552378e6c5ef29850534dfe6c7c8b/_includes/analytics.html#L2).

Comment (waxlamp) [reply mkdocs.yml:69]: > keeping the GA tag in the source code makes it easier if someone wants to spam our GA metrics.

Or someone spins up a copy of the repo and starts accidentally polluting our GA data.

Comment (kabilar) [reply mkdocs.yml:69]: Good point.  I will file some issues in the respective repos.

Comment (aaronkanzer) [reply mkdocs.yml:69]: Pushed https://github.com/dandi/dandi-archive/pull/2126 as a follow-up to this

**Pull request merged**